lib/models/drone.js

```diff
@@ -1,17 +1,35 @@
 'use strict'
 
-var Promise = require('bluebird')
-var mockDrones = []
+var SimpleCollection = require('./simpleCollection.js')
 
-exports.create = function (data) {
-  data.id = 1
-  data.token = 'mocktoken.1'
+let DroneSchema = {
+  'title': 'Generic Drone Schema',
+  'type': 'object',
+  'properties': {
+    'id': { 'type': 'string' }
+  }
+}
```

Comment:
Don't know. On http://json-schema.org/ and similar sites the properties are always in quotes....